README.md

```diff
@@ -1,2 +1,70 @@
-# Hello-Microverse
-Hello Microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](./app_Hello_Microverse_first.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```

Comment:
- Fix your linter issue in your ``PULL REQUEST``.

![Screen Shot 2021-11-23 at 6 51 01](https://user-images.githubusercontent.com/22258274/143027608-cef41c31-ce15-4697-8afa-5c42647091eb.png)



- Check the details and fix the issue.